Update Information Term (#42)
IETF-OPSAWG-WG/draft-opsarea-rfc5706bis: draft-opsarea-rfc5706bis.md

   counter. Then, whether the counter is gathered via SNMP, a CLI
   command, or a syslog message, the counter will have the same meaning.

>    Protocol designers should consider what operational, configuration,

Reviewer: ```suggestion
   Protocol designers must consider what operational, configuration,
```

Per Adrian's comment in #33.

   state, or statistical information will be relevant for effectively

>    monitoring, controlling, or troubleshooting a protocol and its

Author: ```suggestion
   monitoring, controlling, or troubleshooting a new protocol and its
```

>    monitoring, controlling, or troubleshooting the new protocol or its

Reviewer: ```suggestion
   monitoring, controlling, or troubleshooting a protocol or its
```

>    monitoring, controlling, or troubleshooting the new protocol or its

Reviewer 2: Small nit.  What about, "or troubleshooting the new protocol and its"

   extensions. This includes identifying key parameters that reflect the
   protocol’s behavior, performance metrics, error indicators, and any

>    contextual data that would aid in diagnostics or lifecycle management.

Reviewer: ```suggestion
   contextual data that would aid in diagnostic, troubleshooting, or lifecycle management.
```
